sys-apps/sydbox/sydbox-1.0.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="ptrace-based sandbox"
+HOMEPAGE="https://git.exherbo.org/sydbox-1.git"
+SRC_URI="https://git.exherbo.org/sydbox-1.git/snapshot/${PN}-1-${PV}.tar.xz"
```

Comment:
The correct location seems to be `https://dev.exherbo.org/distfiles/${PN}/${P}.tar.bz2`.

Remember to also drop the `S` override when using the official tarball.